Example/Tests/CDTQQuerySqlTranslatorTests.m

```diff
@@ -652,6 +652,21 @@
                 expect(parts.sqlWithPlaceholders).to.equal(@"(\"name\" IS NULL)");
             });
         });
+        
+        describe(@"when using the $in operator", ^{
+            it(@"constructs valid WHERE clause when using a single element in an array", ^{
+                CDTQSqlParts *parts = [CDTQQuerySqlTranslator
+                    wherePartsForAndClause:@[@{ @"name" : @{ @"$in" : @[ @"mike" ]} }]
+                                usingIndex:@"named"];
+                expect(parts.sqlWithPlaceholders).to.equal(@"\"name\" IN ( ? )");
+            });
+            it(@"constructs valid WHERE clause when using multiple elements in an array", ^{
+                CDTQSqlParts *parts = [CDTQQuerySqlTranslator
+                    wherePartsForAndClause:@[@{ @"name" : @{ @"$in" : @[ @"mike", @"fred" ]} }]
+                                usingIndex:@"named"];
+                expect(parts.sqlWithPlaceholders).to.equal(@"\"name\" IN ( ?, ? )");
+            });
```

Comment:
I think you should be checking for correct placeholder values in the Parts object.